CMakeLists.txt

```diff
@@ -124,6 +124,13 @@ endif()
 
 set(ZIG_PIE off CACHE BOOL "produce a position independent zig executable")
 
+# Detect system c++ name
+if ("c++" IN_LIST _CMAKE_CXX_IMPLICIT_LINK_LIBRARIES)
+    set(ZIG_SYSTEM_LIBCXX "c++")
```

Comment:
Can we make it a cached string variable? Then I can also specify it explicitly like `cmake -DZIG_SYSTEM_LIBCXX=c++`.